tools/build_framework.sh

```diff
@@ -13,8 +13,9 @@ usage() {
 # Optional envvars:
 #   REPO_ROOT_DIR: path to root of source repository (default: parent directory of this file)
 #   REPO_NAME: name of the source repository (default: directory name of REPO_ROOT_DIR)
-#   BOOTSTRAP_DIR: path to bootstrap tool, or an empty string to disable bootstrap build (default: <REPO_ROOT_DIR>/sdk/bootstrap)
-#   CLI_DIR: path to the CLI directory, or an empty string to disable CLI build (default: </absolute/framework/path>/cli/)
+#   BOOTSTRAP_DIR: path to bootstrap tool, or "disable" to disable bootstrap build (default: <REPO_ROOT_DIR>/sdk/bootstrap)
```

Comment:
(The issue encountered here: treating empty string as 'disabled' just leads to the following code using the default path -- because empty string is equivalent to unset)